src/api/v1/hooks/onPostHandler.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * onPostHandler Lifecycle hook
+ *
+ * This hook runs immediately after the handler returns, and before
+ * any validation of the response is performed.
```

Comment:
Where is validation on the response performed?

Reply:
Inside hapi